data/images/objects/fallblock/cave-5x7.sprite

```diff
@@ -0,0 +1,7 @@
+(supertux-sprite
+  (action
+    (name "normal")
+    (hitbox 0 32 160 224)
+    (images "cave-5x8.png")

```

Comment:
I have no idea how this works, but I noticed that in every other `.sprite` you added, this line matches the file name, but not here...

```suggestion
    (images "cave-5x7.png")
```

Or maybe this file should be called `cave-5x8.sprite` instead?

Reply:
It is fixed.